server/services/participants.js

```diff
@@ -130,6 +117,44 @@ const getParticipantByID = async (participantInfo) => {
   return participant;
 };
 
+const withdrawParticipant = async (participantInfo) => {
+  const participant = { ...participantInfo };
+  const newHistory = {
+    timestamp: new Date(),
+    changes: [],
+  };
+
+  newHistory.changes.push({
+    field: 'interested',
+    from: participant.interested || 'yes',
+    to: 'withdrawn',
+  });
+  participant.history = participant.history ? [newHistory, ...participant.history] : [newHistory];
+
+  // eslint-disable-next-line no-use-before-define
+  return updateParticipant(participant);
```

Comment:
is there a reason we're disabling the eslint flag rather than putting withdrawParticipant underneath updateParticipant?

Reply:
no particular reason, I will change.